core-services/prow/03_deployment/admin_config_updater_rbac.yaml

```diff
@@ -680,3 +680,42 @@ items:
   - kind: ServiceAccount
     name: config-updater
     namespace: ci
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: config-updater
+    namespace: ocp-private
+  rules:
+    - apiGroups:
+        - rbac.authorization.k8s.io
+        - authorization.openshift.io
+      resources:
+        - rolebindings
+        - roles
```

Comment:
What is the reason why we should deviate from our practice of treating RBAC resources as admin resources and allow config-updater to manage them in this specific case? I'd rather continue to be on the safe side until we decide to change that policy in general.

Reply:
But we need those in order to allow `config-updater` to create the roles in the `ocp-private` namespace. Those are the kind of the rbacs that we consider are admin and apply them manually. Where is the best place for this to live?

Reply:
In all other cases we don't allow `config-updater` to manipulate with rbacs. rbacs are usually placed in `admin_$SOMETHING.yaml` files the appropriate `core-services` subdir and applied manually by `make all-admin` (which will only work if you have rights to impersonate as `system:admin`)